lib/osf-components/addon/components/node-description/styles.scss

```diff
@@ -2,3 +2,25 @@
     word-break: break-word;
     margin-bottom: 15px;
```

Comment:
```suggestion
```
Minor thing: This `margin-bottom` doesn't do anything when the description is truncated (because the heading below has a larger `margin-top`), but it pushes the "see more" button down so it's closer to the heading below than the description text:
![Screen Shot 2019-07-15 at 15 23 59](https://user-images.githubusercontent.com/6776190/61242822-ade28980-a714-11e9-9f9b-b24b82746a22.png)